types/luxon/index.d.ts

```diff
@@ -589,4 +589,6 @@ export class FixedOffsetZone extends Zone {
 
 export class InvalidZone extends Zone { }
 
-export class LocalZone extends Zone { }
+export class LocalZone extends Zone {
+    static get instance(): LocalZone;
```

Comment:
Make compatible with target < ECMAScript 5:
```suggestion
    static readonly instance: LocalZone;
```